docs/source/markdown/podman.1.md

```diff
@@ -68,6 +68,12 @@ pass phrase for SSH identity file
 
 Log messages above specified level: debug, info, warn, error (default), fatal or panic (default: "error")
 
+**--max-workers**=**num**
+
+Set the maximum number of threads Podman will use for executing parallel operations.
+Some Podman operations (among others, stopping and removing containers) will, when given more than one container, work on several containers at a time (one per thread) due to long delays inherent in stopping containers.
+The default is 1 plus the number of CPUs available on the system multiplied by 3.
+
```

Comment:
If we end up hiding this, could we capture this information somewhere?  Perhaps a code comment in root.go?

Reply:
Actually part of this is down below in parallel.go, if we could beef up the code comments there with this, that would be great.